mobile/mobile_12_2.asm

```diff
@@ -10,47 +12,60 @@ MobileCheckOwnMonAnywhere:
 	ld e, 0
 	ld hl, wPartyMon1Species
 	ld bc, wPartyMonOTs
-.asm_4a851
+
+	; Run .CheckMatch on each Pokémon in the party.
+.partymon
 	call .CheckMatch
-	ret c
+	ret c ; found !
+
 	push bc
 	ld bc, PARTYMON_STRUCT_LENGTH
 	add hl, bc
 	pop bc
 	call .AdvanceOTName
 	dec d
-	jr nz, .asm_4a851
+	jr nz, .partymon
+
+	; Run .CheckMatch on each Pokémon in the PC.
 	ld a, BANK(sBoxCount)
 	call OpenSRAM
 	ld a, [sBoxCount]
 	and a
-	jr z, .asm_4a888
+	jr z, .boxes
+
 	ld d, a
 	ld hl, sBoxMon1Species
 	ld bc, sBoxMonOTs
-.asm_4a873
+.openboxmon
 	call .CheckMatch
-	jr nc, .asm_4a87c
+	jr nc, .loop
+
+	; found!
```

Comment:
Superfluous comment again. Should be obvious from function structure and label names. Same for all the other "found !" comments.